src/ui/Controllers/HomeController.cs

```
@@ -17,10 +18,22 @@ public HomeController(ManageApi manageApi)
         }
 
         // GET: Home
+        [Authorize]
         public ActionResult Index()
         {
+            var orgs = _manageApi.GetOrganisations().Result.ToList();
+
+            if (orgs.Count() == 1)
+            {
+                return this.RedirectToAction("Index", "Courses", new {organisationId = orgs[0].OrganisationId });
+            }
+
+            if (orgs.Count() > 1)
+            {
+                return this.RedirectToAction("Index", "Organisations");
+            }
 
-            return this.RedirectToAction("Index", "Courses");
+            return null;
```

I assume this is the failure case in which case it should throw an exception